packages/core/template/portfolio/contents/experience.md

```diff
@@ -0,0 +1,29 @@
+<frontmatter>
+  title: Experience
+</frontmatter>
+
+<br>
+
+# Experience
+
+<panel type="seamless" header="Experience 1" no-close minimized>
+  <md>**Description of experience (Name of role, duration, etc.)**
```

Comment:
maybe you could have a [badge](https://markbind.org/userGuide/components/presentation.html#tabs) with the date since normally you would list expereinces in chronological order

Reply:
This could be even better with a full blown example of the experience 
I was thinking like 
```
**Company**: Full stack engineer
From {date badge} to {date badge}

Technical skills: 
- ....
```

What do you think